js/src/index.js

```diff
@@ -45,6 +45,10 @@ function NSFWFilePoller(watchPath, eventCallback, debounceMS) {
   this.stop = async () => {
     clearInterval(filePollerInterval);
   };
+
+  // Because I'm not that concerned about pausing a single-file watcher
+  this.pause = () => Promise.resolve();
```

Comment:
It might be better to hook this up to `stop` instead.

Reply:
That also makes sense